app/javascript/components/organization-teams-dropdown.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <teams-dropdown
+    :github-login="githubLogin"
+    :teams="this.organizationTeams">
+  </teams-dropdown>
+</template>
+
+<script>
+import TeamsDropdown from './teams-dropdown';
+import { mapState } from 'vuex';
```

Comment:
Absolute imports should come before relative imports  import/imports-first

Reply:
Qué raro esto, si `vuex` es absoluto...